plugins/core/IO/qLASIO/include/LasScalarFieldSaver.h

```diff
@@ -74,6 +74,16 @@ class LasScalarFieldSaver
 			*reinterpret_cast<T*>(dest) = static_cast<T>(value);
 		}
 	}
+	template <typename T>
+	static void WriteScalarValueAsFltDbl(ScalarType value, uint8_t* dest)
+	{
+		if (value > std::numeric_limits<T>::max())
+			*reinterpret_cast<T*>(dest) = std::numeric_limits<T>::max();
+		else if (value < -std::numeric_limits<T>::max())
+			*reinterpret_cast<T*>(dest) = -std::numeric_limits<T>::max();
+		else
+			*reinterpret_cast<T*>(dest) = static_cast<T>(value);
+	}
```

Comment:
I would suggest changing `std::numeric_limits<T>::min()` to `std::numeric_limits<T>::lowest()` in the `WriteScalarValueAs` function it would help us avoid duplicating

Reply:
you mean `-std::numeric_limits<T>::max()` by `std::numeric_limits<T>::lowest()` I guess?